### PR TITLE
Update environment-compatibility.md

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/setup/environment-compatibility.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ As WSO2 Identity Server is a Java application, you can generally run them on mos
 		<th>Supported JDK versions</th>	
 		<td>
 			<ul>
-				<li>CorrettoJDK 8</li>
-				<li>AdoptOpenJDK 8</li>
-				<li>AdoptOpenJDK 11</li>
+				<li>JDK 11.0.8</li>
+				<li>JDK 11.0.9</li>
+				<li>OpenJDK 1.8</li>
```
Comment (NilukaSripalim) [inline en/docs/setup/environment-compatibility.md:15]: OpenJDK 1.8 & OpenJDK 8 is same, then lets keep one

File: en/docs/setup/environment-compatibility.md [hunk 2]
```diff
@@ -112,15 +111,18 @@ WSO2 Identity Server supports the following DBMSs.
 ### Tested Web Browsers
 
 <ul>
-<li>Chrome Version 78.0.3904.87 (Official Build) (64-bit)</li>
-<li>Firefox 70.0.1 (64-bit)</li>
-<li>Internet Explorer 11.576.14393</li>
-<li>Safari 13.0.3</li>
```
Comment (NilukaSripalim) [inline en/docs/setup/environment-compatibility.md:118]: please keep safari

File: en/docs/setup/environment-compatibility.md [hunk 3]
```diff
@@ -112,15 +111,18 @@ WSO2 Identity Server supports the following DBMSs.
...
+	<li>Mozilla Firefox 75.0 (64-bit)</li>
+	<li>Mozilla Firefox 81.0.2 (64-bit)</li>
+	<li>Google Chrome Version 84.0.4147.125 (Official Build) (64-bit)</li>
```
Comment (nilminiwso2) [inline en/docs/setup/environment-compatibility.md:116]:
```suggestion
	<li>Google Chrome Version 84.0.4147.125 (Official Build, 64-bit)</li>
```

File: en/docs/setup/environment-compatibility.md [hunk 4]
```diff
@@ -112,15 +111,18 @@ WSO2 Identity Server supports the following DBMSs.
...
+	<li>Google Chrome Version 85.0.4183.102 (Official Build) (64-bit)</li>
```
Comment (nilminiwso2) [inline en/docs/setup/environment-compatibility.md:117]:
```suggestion
	<li>Google Chrome Version 85.0.4183.102 (Official Build, 64-bit)</li>
```

File: en/docs/setup/environment-compatibility.md [hunk 5]
```diff
@@ -112,15 +111,18 @@ WSO2 Identity Server supports the following DBMSs.
...
+	<li>Google Chrome Version 86.0.4240.75 (Official Build) (x86_64)</li>
```
Comment (nilminiwso2) [inline en/docs/setup/environment-compatibility.md:118]:
```suggestion
	<li>Google Chrome Version 86.0.4240.75 (Official Build, x86_64)</li>
```

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/setup/environment-compatibility.md [hunk 6]
```diff
@@ -112,15 +111,18 @@ WSO2 Identity Server supports the following DBMSs.
...
+	</li>MS Edge 44.18362</li>
 </ul>
 
 ### Tested LDAPs
 
-Following is a list of LDAPs that WSO2 Identity Server 5.9.0 is tested with.
+Following is a list of LDAPs that WSO2 Identity Server 5.11.0 is tested with.
 
 <ul>
 	<li>Open LDAP 2.4.28</li>
```
Comment (NilukaSripalim) [inline en/docs/setup/environment-compatibility.md:128]: Please Add Microsoft Active Directory Windows 2012

Comment (gomathyk) [reply en/docs/setup/environment-compatibility.md:128]: This isn't removed.

**Review by NilukaSripalim — APPROVED** [review]

**Pull request merged**